lib/Net/ASN.pm

```diff
@@ -325,6 +325,17 @@ sub gettype () {
 	return ($informat);
 }
 
+##RFC 6996 reserves some ASN's for private use
+sub is_private {
```

Comment:
Perhaps drop the underscore, since the convention in the code seems that I don't use it in subroutine names other than to start the name of a private method